ui/client/components/toolbars/creator/ComponentIcon.tsx

```diff
@@ -17,13 +17,14 @@ export const getComponentIconSrc = memoize((node: NodeType, processDefinitionDat
   if (node) {
     const nodeComponentId = ProcessUtils.findNodeConfigName(node)
     const componentConfig = processDefinitionData.componentsConfig?.[nodeComponentId]
-    const iconFromConfig = componentConfig?.icon
-    const iconBasedOnType = node.type && `${node.type}.svg`
+
+    const iconFromConfig = componentConfig?.icon && absoluteBePath(componentConfig.icon)
+    const iconBasedOnType = node.type && absoluteBePath(`/assets/components/${node.type}.svg`)
```

Comment:
why call `absoluteBePath` twice?

Reply:
It doesn't matter.. It will be done once.. But, ok I can do a little change calling absolutePath once. IMO right now is clearer and there is less code.